spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -402,14 +404,16 @@ def not_ok?; false; end
               if order.current == :second
                 :second_access
               else
-                order.pass_to :second, :resume_on => :sleep
```

Comment:
You'll only see this "possible deadlock" error for non-deadlock situations where threads are interleaved when multiple threads are accessing the same uninitialized `let` (or a `let` inside of a `subject` from another thread), which seems like it'd be rare but obviously up to y'all.